src/configure/helper/commonHelper.ts

```diff
@@ -63,4 +67,36 @@ export function stringCompareFunction(a: string, b: string): number {
         return 1;
     }
     return 0;
+}
+
+export async function executeFunctionWithRetry(func: () => Promise<any>, retryCount: number = 100, retryTimeoutInSec: number = 300, retryIntervalTimeInSec: number = 2, errorMessage?: string): Promise<any> {
+    let internalErrorMessage = '';
+    let functionExecutionPromise = Q.Promise(async (resolve, reject) => {
+        let result = null;
+        for (;retryCount > 0; retryCount--) {
+            try {
+                result = await func();
+                resolve(result);
```

Comment:
if you directly do `return resolve(result)` then you might not need to break and handle it again in L91.

Reply:
But not sure, you can try